wagtail/admin/static_src/wagtailadmin/scss/layouts/login.scss

```diff
@@ -66,6 +66,10 @@ input[type=checkbox]:before {
     border: 1px solid #555;
 }
 
+.fields-wrapper {
```

Comment:
This ensures the positioning remains correct when the 'incorrect username and password' message shows.